string before
string after
geometry_msgs/Pose before_pose
---
geometry_msgs/PoseStamped after_pose
